openmetadata-ui/src/main/resources/ui/src/components/common/RichTextEditor/RichTextEditorPreviewer.tsx

```diff
@@ -110,7 +111,11 @@ const RichTextEditorPreviewer = ({
       data-testid="viewer-container"
       dir={i18n.dir()}>
       <div
-        className={classNames('markdown-parser', textVariant)}
+        className={classNames(
+          'markdown-parser',
+          textVariant,
+          readMore ? '' : reducePreviewLineClass
+        )}
```

Comment:
Since it is a widely used common component did we check its effect everywhere?

Reply:
`reducePreviewLineClass` is a prop, and will only affect is we pass it!

Reply:
In that case, can we default its value to an empty string?

Reply:
since it is used within `classNames` it will automatically scrap it, in case of undefined!